src/drivers/serac.cpp

```diff
@@ -232,9 +232,9 @@ int main(int argc, char* argv[])
     last_step = (t >= t_final - 1e-8 * dt);
   }
 
-  serac::output::outputSummary(datastore, serac::StateManager::collectionName(), output_directory);
```

Comment:
The setting of this variable got removed in #597. I decided instead of re-adding that line to just make the file more generically named due to it living in a directory we create.

`serac_datacoll_summary.json` -> `_summary.json` -> `summary.json`